core/store/src/trie/mem/iter.rs

```diff
@@ -70,16 +72,18 @@ impl<'a> Crumb<'a> {
 /// The trail and the key_nibbles may have different lengths e.g. an extension trie node
 /// will add only a single item to the trail but may add multiple nibbles to the key_nibbles.
 
-pub struct MemTrieIterator<'a> {
-    root: Option<MemTrieNodePtr<'a, STArenaMemory>>,
+pub type STMemTrieIterator<'a> = MemTrieIterator<'a, STArenaMemory>;
```

Comment:
I'm thinking of reverting this nomenclature back to `MemTrieIterator`

Reply:
There'll be a HybridMemTrieITerator that replaces this I assume?